src/driver/linux_resource/kernel_compat_funcs.sh

```diff
@@ -547,14 +547,25 @@ vmsg "KPATH      := $KPATH"
 #  CONFIG_PTP_1588_CLOCK: PTP clock support
 
 [ -n "${ARCH:-}" ] && export ARCH
-eval $(read_make_variables KBUILD_SRC ARCH SRCARCH CONFIG_X86_32 CONFIG_X86_64 CONFIG_PTP_1588_CLOCK abs_srctree)
+eval $(read_make_variables KBUILD_SRC ARCH SRCARCH CONFIG_X86_32 CONFIG_X86_64 CONFIG_PTP_1588_CLOCK)
+
+# linux-6.9: $abs_srctree is unexported. It breaks the code below on Debian,
+# due to its kernel headers path split for "arch" and "common" ones.
+# Use $srctree with KBUILD_ABS_SRCTREE=1 instead.
+eval $(EXTRA_MAKEFLAGS="$EXTRA_MAKEFLAGS KBUILD_ABS_SRCTREE=1" read_make_variables srctree)
```

Comment:
Did you try merging this with the previous `read_make_variables` (i.e. both would run with `KBUILD_ABS_SRCTREE=1`)? Just trying to save a few milliseconds of build time

Reply:
Fixed